google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Subscriber.java

```diff
@@ -500,7 +490,31 @@ Builder setInternalHeaderProvider(HeaderProvider internalHeaderProvider) {
       return this;
     }
 
-    /** Sets the flow control settings. */
+    /**
+     * Sets the flow control settings.
+     *
+     * <p>In the example below, the {@Subscriber} will make sure that
+     *
+     * <ul>
+     *   <li>there are at most ten thousand outstanding messages, and
+     *   <li>the combined size of outstanding messages does not exceed 1GB.
+     * </ul>
+     *
+     * "Outstanding messages" here means the messages that have already been given to {@link
+     * MessageReceiver} but not yet {@code acked()} or {@code nacked()}.
+     *
+     * <pre>{@code
+     *    FlowControlSettings flowControlSettings =
```

Comment:
nit: the spacing is a bit awkward in this pre block - could it be made consistent?

Reply:
Ah thank you for catching!